Source/UltraleapTrackingCore/Private/LeapComponent.cpp

```diff
@@ -388,8 +388,6 @@ void ULeapComponent::OnDeviceAdded(IHandTrackingWrapper* DeviceWrapper)
 	{
 		RefreshDeviceList(true);
 	}
-	ActiveDeviceSerial = DeviceWrapper->GetDeviceSerial();
```

Comment:
Why was this line deleted?

Reply:
it's a fix to pinch and grab when openXR is not available: https://ultrahaptics.atlassian.net/browse/UNREAL-729

Reply:
Ok, thanks.